### ISSUE TITLE
Make spells markable as domain spells

### ISSUE THREAD

**Issue by ekrebs5** [roc-and-roll/roc-and-roll#211]

**Comment by corinnaj** [comment]
Do you have a preference on where and how would you want to do this?
In the spell list for the character? Do you just want a tag or an icon in the spell name? Or something more fancy?

**Comment by ekrebs5** [comment]
Yes, in the spell list would be awesome. A checkbox / a tag would be great, so you don't accidently un-prepare the domain spells.

**Comment by ekrebs5** [comment]
Hm, actually, maybe it's a special state of the prepare button?

**Comment by corinnaj** [comment]
Hmm, so it's not about marking it necessarily, but rather as always having it prepared? Or do you also want to have an icon or tag?

**Comment by s4pu** [comment]
I think all spells that are "always prepared" should never count against prepared spells, but I am not 100% sure if that is true.
